RetrospectiveExtension.Frontend/package.json

```diff
@@ -84,12 +88,18 @@
     "webpack-dev-server": "^4.7.2"
   },
   "jest": {
+    "testPathIgnorePatterns" : [
+      "./components/__tests__/setupTests.tsx"
```

Comment:
prevent the `setupTests` file from being detected as a test by Jest as it will "fail" in testing.